src/utils/SolutionsUtils.js

```diff
@@ -55,9 +55,25 @@ const patchSolutionIfLocalConfigExists = async (originalSolution) => {
   ConfigUtils.patchSolution(originalSolution, overridingSolution);
 };
 
+const fixNotCompatibleValuesInSolution = (solution) => {
```

Comment:
Could you rename this function to `patchIncompatibleValuesInSolution` or `patchInconsistentValuesInSolution` please? :slightly_smiling_face:

Reply:
DONE